src/read.c

```diff
@@ -2838,6 +2838,7 @@ avifResult avifDecoderReset(avifDecoder * decoder)
             // Harvest CICP from the AV1's sequence header, which should be very close to the front
             // of the first sample. Read in successively larger chunks until we successfully parse the sequence.
             static const size_t searchSampleChunkIncrement = 64;
+            static const size_t searchSampleSizeMax = 4096;
```

Comment:
We could also fix this bug by doubling searchSampleSize instead of adding a fixed increment.

Reply:
I think if you're confident that the sequence header needs to be in the front of the AV1 payload that this should be sufficient.

Reply:
The first sequence header OBU must come before any frame. The only OBUs that may come before the first sequence header OBU are:
- the temporal delimiter OBU (very small because it has no payload data)
- padding OBUs
- unlikely but seems to be allowed by the spec: maybe certain metadata OBUs